src/contents/articles/hello-world/index.md

```diff
@@ -1,12 +1,10 @@
 ---
-title: Hello World
+title: Introducing Namespace
 author: Marcus Sanatan
 date: 2015-05-24
 template: article.jade
 ---
 
-### This is **namespace**
-
 Hi everyone, we're officially launching namespace - a club for software developers in Trinidad and Tobago. We're a movement to remove the silos of people scattered throughout the nation that prevents us from build a community. 
```

Comment:
Additionally, I think you would should be direct and say that you want to bring developers together. The 'silo removal' metaphor doesn't make sense to me which means it either doesn't closely mirror the challenges we have or you need to introduce these challenges.